README.md

```diff
@@ -193,11 +193,31 @@ cloud.network.openshift.io/egress-ipconfig: [{"interface": "$IFNAME/$IFID", "ifa
 
 # How to hack/debug
 
+## Run locally
+
 When debugging this component make sure to do the following:
 
 1. Run the `./hack/run-locally.sh`, but make sure you have a `KUBECONFIG`
    initialized in your environment variables. If running against an OpenShift
    cluster: the credentials will be copied locally to your computer. That being
    said, you probably won't be able to talk to the cloud API endpoints from your
    local development environment, so you might need to comment out code which is
-   non-essential to your testing.
\ No newline at end of file
+   non-essential to your testing.
+
+## Run from an new image
```

Comment:
can you add a blurb about what exactly this will actually do? ... patch deployment.apps network-operator with
custom image, etc.

Reply:
done :-)